docs/guide/http_cache.md

```diff
@@ -852,7 +853,7 @@ See [Tagging from Twig Templates](https://foshttpcachebundle.readthedocs.io/en/2
 `ezplatform-http-cache` uses Repository API event subscribers to listen to events emitted on Repository operations,
 and depending on the operation triggers expiry on a specific tag or set of tags.
 
-For example on the move Location event the following tags are purged:
+For example, on the move Location event, the following tags are purged:
```

Comment:
If it is a distinct event name, would it make sense to make "move Location" standa out, like, put it in quotes?